src/cmake/thirdparty/FindMFEM.cmake

```diff
@@ -110,6 +110,17 @@ else()
 
     list(APPEND MFEM_LIBRARIES ${mfem_tpl_lnk_flags})
 
+    # Check if MFEM was built with CUDA
+    if(mfem_cfg_file_txt MATCHES "MFEM_USE_CUDA += YES")
+        if(NOT ENABLE_CUDA)
+            message(WARNING "MFEM was built with CUDA but CUDA is not enabled")
+        endif()
+        list(APPEND MFEM_INCLUDE_DIRS ${CUDA_INCLUDE_DIRS})
```

Comment:
the cuda registered library is added in a lot of places.  Is it possible it was not in a few places due to the new mfem+cuda functionality?  I would prefer that instead of doubling up this information.

Reply:
After discussing with @joshessman-llnl , I think this is fine for now but can be improved upon later down the road.